### ISSUE TITLE
More uniform view for combined multicontent files on MTP responder.

### ISSUE THREAD

**Issue by retro16** [rashevskyv/dbi#125]
Using version 389. I want to use the MTP responder to sanitize my game collection by transfering the game to the switch, then get it back from the "Installed games" directory.

The issue is that DBI offers a nice combined NSP only for games with actual updates or DLCs, games with no updates nor mods aren't present and are found only in the subdirectory. Also, file names use a completely different pattern in subdirectories and in combined form.

That would be very cool to make games without updates appear the same way as combined NSP, with the [B] suffix instead of [B+v65535]. This way, I could upload many games to the switch, then re-download all combined files, making a sanitized collection with uniform naming much easier.

Thank you so much for this excellent program.

**Comment by BlackHoleExelion** [comment]
Good point. My suggestion is to set up 2 folders separately under "Installed games". One for combined games including those without upd and dlc (i.e. no individual game folders, only .NSPs in it); one for game folders with patches (uncombined).
